addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.xml

```diff
@@ -81,9 +81,53 @@
                         t-esc="dashboard.displayName"
                         t-att-data-name="dashboard.displayName"
                         class="o_search_panel_category_value list-group-item cursor-pointer border-0"
-                        t-att-class="{'active': dashboard.id === state.activeDashboard.id}"/>
+                        t-att-class="{'active': dashboard.id === state.activeDashboard?.id}"/>
                 </ul>
             </section>
         </div>
     </t>
+
+    <div t-name="spreadsheet_dashboard.DashboardLandingPage" class="mx-auto h-100" style="max-width:1000px;">
+        <h1 class="text-primary text-center pt-3">Welcome to Dashboards</h1>
+        <h2 class="text-action">What is Dashboards?</h2>
+        <p>
+            Dashboards allows you to access all your important information.
+            Your data is sorted into customizable categories and translated
+            into visual representations for easy comprehension and informed decision-making.
+        </p>
+        <img src="/spreadsheet_dashboard/static/img/dashboards.png" alt="Dashboards" style="max-width:800px;" class="mb-5 mx-auto border shadow"/>
+        <h2 class="text-action pt-3">How do you create a dashboard?</h2>
+        <p>
+            To create or edit a dashboard, you need to have the proper rights (Dashboards: Admin).<br/>
+            Then you can either go to:
+            <ul>
+                <li>Dashboards > Configuration > Select a category > click on Add a spreadsheet > select a Blank spreadsheet</li>
+                <li>Documents > New spreadsheet > File > Add to dashboard > select a category</li>
```

Comment:
I don't know if it was considered but I feel like using 
```html
<span class="fa fa-chevron-right"/>
```
instead of ">". could look nice. Altough requires some tinkering with the font-sze because of `fa`  defaults.

Reply:
Yes I considered it, but it was not straightforward with the font-size, alignment and ...